src/pubsub/mod_pubsub.erl

```diff
@@ -63,6 +63,7 @@
 -define(STDNODE_MODULE, node_flat).
 -define(PEPNODE, <<"pep">>).
 -define(PUSHNODE, <<"push">>).
+-define(NO_DEFAULT_CACHE, no_last_item_cache).
```

Comment:
why actually do we need to define that macro ? Can't we use `no_last_item_cache` directly ?

Reply:
I was using that same atom in a couple of places already, dunno, should I remove the macro?